IBM/carbon-addons-iot-react
# fix(table): added overflow scroll and text nowrap

Closes #824

Fixes: 
![Screen Shot 2020-01-15 at 1 59 56 PM](https://user-images.githubusercontent.com/25177649/72466874-601c0c00-379f-11ea-871f-1d447d8178cc.png)

![Screen Shot 2020-01-15 at 2 00 06 PM](https://user-images.githubusercontent.com/25177649/72466876-614d3900-379f-11ea-82fe-2d5532488c21.png)


**Summary**

- Wrapped the table in a div with styling to handle overflow in Table
- Added styling to nowrap the text as it got messed up when overflow was fixed
- Updated snapshots

**Change List (commits, features, bugs, etc)**

- src/components/Table/Table.jsx
- src/components/Table/_table.scss

**Acceptance Test (how to verify the PR)**

1. Go to Simple Stateful Table example
2. Reduce window size until a few columns are hidden
3. Scroll to the right on table
4. See that there are no longer styling overflow issues
